### PR TITLE
Add Utils::calculateBitmask method

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

File: Symfony/CS/Tokenizer/Token.php [hunk 1]
```diff
@@ -441,14 +443,10 @@ public function toArray()
 
     public function toJSON()
```
Comment (GrahamCampbell) [inline Symfony/CS/Tokenizer/Token.php]: Should be named toJson() shouldn't it?

Comment (keradus) [inline Symfony/CS/Tokenizer/Token.php]: Not sure about it.
It's already inside btw, I didn't change it.

Comment (GrahamCampbell) [inline Symfony/CS/Tokenizer/Token.php]: Renaming it isn't a bc break i don't think given that php isn't case sensitive with function calls.

Comment (keradus) [inline Symfony/CS/Tokenizer/Token.php]: Done

**Comment by keradus** [comment]
PR has been merged.

**Pull request closed without merging**